home/home_2.1/compile.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+clang++ -Wall -Wextra -Wpedantic -Werror -std=c++23 "$@"
```

Comment:
not sure whether we need "" for $@ as at least in some shells it may be treated as a single argument while our intention is to forward argument passing.